itkwidgets/standalone/index.html

```diff
@@ -0,0 +1,98 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- <link rel="icon" type="image/png" sizes="50x50" href="/itk-vtk-viewer/logo.png"> -->
+    <!-- <link rel="apple-touch-icon" type="image/png" sizes="50x50" href="/itk-vtk-viewer/logo.png"> -->
+
+    <script src="https://cdn.jsdelivr.net/npm/imjoy-rpc@0.5.21/dist/hypha-rpc-websocket.min.js"></script>
+    <link rel="icon" type="image/png" href="static/favicon-32x32.png">
+  </head>
+
+  <body>
+    <div class="content" style="position: absolute; width: 100vw; height: 100vh; top: 0; left: 0; overflow: hidden; background: black; margin: 0; padding: 0;"></div>
+    <script type="text/javascript" src="https://bafybeiauuswm657tct7b7kpnorrsz7hvvaljpsbeuw35hkfobtyyrbblpm.on.fleek.co/itkVtkViewer.js">    </script>
+    <script>
+    async function setupHypha(config) {
+        globalThis.config = config
+        const url = config.server_url;
+        const extraHeaders = {};
+        if (config.token) {
+          extraHeaders.Authorization = "Bearer " + config.token;
+        }
+        // Note: extraHeaders only works for polling transport (the default)
+        // If we switch to websocket only, the headers won't be respected
+        if(globalThis.server){
+          globalThis.server.disconnect();
+        }
+        const server = await hyphaWebsocketClient.connectToServer(config)
+        return server
+     }
+     async function setupViewerForImJoy(server) {
+      const version = itkVtkViewer.version
+      await server.registerService({
+        name: 'itkwidgets-service',
+        id: 'itkwidgets_service',
+        config: { visibility: "public" },
+        version,
+        description: '2D / 3D web image, mesh, and point set viewer using itk-wasm and vtk.js ',
+        type: 'window',
+        docs: 'https://kitware.github.io/itk-vtk-viewer/docs/imjoy.html',
+        authors: ['Matt McCormick', 'Wei Ouyang'],
+        license: '3-Clause BSD License',
+        labels: ['visualization', 'itk', 'vtk', 'image', 'mesh', 'geometry', 'point set', '2D', '3D'],
+        icon: 'https://kitware.github.io/itk-vtk-viewer/app/favicon-32x32.png',
+        cover: ['https://kitware.github.io/itk-vtk-viewer/docs/howToUse.jpg', 'https://kitware.github.io/itk-vtk-viewer/docs/imjoy.png'],
+
+      });
+      itkVtkViewer.imJoyCodecs.forEach((codec) => {
+        server.registerCodec(codec)
+      })
+      const remoteID = `${workspace}/itkwidgets-server`;
+      const remoteService = await server.getService(`${remoteID}:itkwidgets-input-obj`);
+      const data = await remoteService.inputObject();
+      const viewer = await itkVtkViewer.createViewer(container, data.data);
+      await server.registerService({
+        name: "itk_vtk_viewer",
+        id: "itk-vtk-viewer",
+        description: "Retrieve the itk-vtk-viewer object",
+        config: {
+          visibility: "protected",
+          requireContext: false,
+          runInExecutor: false,
```

Comment:
```suggestion
          run_in_executor: false,
```

These config are in snake_case: https://github.com/imjoy-team/imjoy-rpc/blob/43c48a13840107f6a5c3964b8f0f062159964c83/javascript/src/hypha/rpc.js#L405-L410

Reply:
Fixed, thank you!!